source/contracts/reporting/ReportingToken.sol

```diff
@@ -102,13 +102,16 @@ contract ReportingToken is DelegationTarget, Typed, Initializable, VariableSuppl
 
     // NOTE: UI should warn users about calling this before first calling `migrateLosingTokens` on all losing tokens with non-dust contents
     // NOTE: we aren't using the convertToAndFromCash modifier here becuase this isn't a whitelisted contract. We expect the reporting window to handle disbursment of ETH
-    function redeemWinningTokens() public afterInitialized returns (bool) {
+    // CONSIDER: If all markets are finalized should we allow forgoing fees?
```

Comment:
Meh, whatever is easier.  There is no reason someone would other than on accident, and the UI can help protect users from that.  There is also no problem with someone skipping fees if they like.